src/main/java/io/github/genomicdatainfrastructure/daam/services/SubmitApplicationService.java

```diff
@@ -37,6 +39,12 @@ public void submitApplication(Long id, String userId) {
                 .applicationId(id)
                 .build();
 
-        remsApplicationCommandApi.apiApplicationsSubmitPost(remsApiKey, userId, command);
+        SubmitApplicationResponse response = remsApplicationCommandApi.apiApplicationsSubmitPost(
```

Comment:
hint: next time you can use `var` instead of repeating the type in the variable declaration. Nowadays the type is only required in a few scenarios, like method/function signatures.